Codo-Baggins/game_crave
# Feature/router

## Is this a fix or feature?
-Feature

## What is the change?
- Allows users to navigate between pages using React Router

## Where should the reviewer start?
- Lines 69 - 106 in App.js and then look at the Router Link tags in Form.js and GameCard.js

## How should this be tested?
- Navigate to the application
- Type the name of a video game such as "pacman" into the input field on the page
- Notice how the URL changes and the content on the page renders
- Click the "Add To WishList" button
- Then click the "My Wish List" button at the top of the page
- A user will be taken to the Wish List page where they can click on a video game to view its details or they can search for a new game by typing into the input field again
